### DISCUSSION TITLE
Garena Free fire API

### DISCUSSION THREAD

**Discussion by CebolinhaWiz** [frontendbr/forum#2313] [Perguntas e Respostas]
Como faço para me conectar no jogo Free Fire através de um bot do discord?

**Comment by felquis** [comment]
Também quero saber

**Comment by jeeanribeiro** [comment]
Não deu pra entender muito bem o que vc quer, consegue dar um exemplo?

**Reply by Draken119** [reply]
Ele quer um bot de discord que loga no free fire e entra num grupo do jogo e sai

**Comment by BBCORHitter** [comment]
você pode encontrar algumas informações válidas em [https://freefirereadvanceservers.net/](https://freefirereadvanceservers.net/) e facilmente conectar o discord bot à sua conta do Free Fire.

**Comment by LuizGustavo536** [comment]
Tá funcionando ainda?

**Reply by CebolinhaWiz** [reply]
Eu consegui criar um sistema que busca informações dos jogadores e das guildas

**Reply by ryzenxff** [reply]
como?

**Reply by CebolinhaWiz** [reply]
eu fiz uma api não oficial, se quiser saber mais me chama no telegram, @B10_NET

**Comment by ffmania** [comment]
Você fala de API igual ao desse site?

https://www.freefiremania.com.br/data-de-criacao-conta-free-fire.html

**Reply by CebolinhaWiz** [reply]
sim

**Comment by marquinhoslimaa2012-jpg** [comment]
Alguém pode me ajudar? Queria criar um bot que crie salas no free fire automaticamente por um painel do Discord alguém consegue ajudar?

**Reply by soaresalmeidadasilvajoaogabrie-ui** [reply]
eu

**Comment by Brenno12** [comment]
Eu gostaria de um API para sala bot para criar sala automática no discord

**Reply by soaresalmeidadasilvajoaogabrie-ui** [reply]
to atras tmb
